ansible_collections/arista/avd/roles/eos_designs/docs/tables/network-services-l2vlans-settings.md

```diff
@@ -70,7 +70,7 @@
             # Tags leveraged for networks services filtering.
             # Tags are matched against filter.tags defined under node type settings.
             # Tags are also matched against the node_group name under node type settings.
-            tags:
+            tags: # default=['all']
```

Comment:
we are losing information here that this is a type str

Reply:
It is on the line below.